components/script/dom/gpucomputepipeline.rs

```diff
@@ -76,7 +76,12 @@ impl GPUComputePipeline {
     ) -> WebGPUComputePipeline {
         let compute_pipeline_id = device.global().wgpu_id_hub().create_compute_pipeline_id();
 
-        let (layout, implicit_ids, _) = device.get_pipeline_layout_data(&descriptor.parent.layout);
+        let layout = device
+            .get_pipeline_layout_data(&descriptor.parent.layout)
+            .explicit();
+        let implicit_ids = device
+            .get_pipeline_layout_data(&descriptor.parent.layout)
+            .implicit();
```

Comment:
```suggestion
        let pipeline_layout = device.get_pipeline_layout_data(&descriptor.parent.layout);
```
and then do implicit, explicit method calls where this is actually needed.

Reply:
also same for render pipeline

Reply:
okay, let me fix that